packages/esm-tb-app/src/views/program-management/tb-program-management-config.json

```diff
@@ -0,0 +1,127 @@
+{
+    "menuId": "programManagement",
+    "tabDefinitions": [
+      {
+        "tabName": "TB/MDR TB Enrolment",
+        "headerTitle": "TB/MDR TB Enrolment",
+        "displayText": "TB/MDR TB Enrolment",
+        "encounterType": "9a199b59-b185-485b-b9b3-a9754e65ae57",
+        "columns": [
+          {
+            "id": "enrollmentDate",
+            "isDate": true,
+            "title": "Enrollment Date",
+            "concept": "161552AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+          },
+          {
+            "id": "caseID",
+            "title": "Case ID",
+            "concept": "162576AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+          },
+          {
+            "id": "tbTreatmentId",
+            "title": "TB Treatment ID",
+            "concept": "treatmentId"
+          },
+          {
+            "id": "treatmentStartDate",
+            "isDate": true,
+            "title": "Treatment Start Date",
+            "concept": "tBTreatmentStartDateConcept"
```

Comment:
Is this working? it is not a uuid, I know we are moving to this implementation next but I am not sure it works now.